phoenix-core/src/test/java/org/apache/phoenix/mapreduce/index/BaseIndexTest.java

```diff
@@ -36,7 +36,7 @@
  * Creates a simple data table and index table
  *
  */
-public class BaseIndexTest extends BaseConnectionlessQueryTest {
+public class  BaseIndexTest extends BaseConnectionlessQueryTest {
```

Comment:
Btw somehow this change also didn't trigger end2end tests. Perhaps adding new line in phoenix-core/pom.xml would do.